### PR TITLE
NP-6561 - csv upload for companybranddao

### PR DESCRIPTION
**Pull request by tala-abuadas**
https://nanopay.atlassian.net/browse/NP-6561

### CODE REVIEW

File: src/files.js [hunk 1]
```diff
@@ -1205,5 +1205,7 @@ FOAM_FILES([
   { name: "foam/nanos/ndiff/NDiffJournal", flags: [ "java" ] },
   { name: "foam/nanos/ndiff/NDiffRuntimeDAO", flags: [ "java" ] },
   { name: "foam/u2/view/ComparisonView" },
-  { name: "foam/u2/view/SuggestedTextField", flags: [ "web" ] }
+  { name: "foam/u2/view/SuggestedTextField", flags: [ "web" ] },
+  { name: "foam/demos/csv/CsvUploadDemo", flags: ["web"] }
```
Comment (sarthak-marwaha) [inline src/files.js]: Are we adding demo files to files.js as well? @kgrgreer

**Review by sarthak-marwaha — COMMENTED** [review]
Everything else LGTM

File: src/files.js (hunk 1 above)
Comment (kgrgreer) [reply src/files.js]: Yes, please add "demo" flag to flags: for demos. Then we should not include these in production builds.

**Comment by kgrgreer** [comment]
LGTM, just add the "demo" flag where noted.

**Pull request merged**